src/App.jsx

```diff
@@ -1,7 +1,5 @@
 import { useState } from 'react'
```

Comment:
can you create your components for testing styling purpose not adding your styles into the App.jsx